geomesa-accumulo/geomesa-accumulo-tools/src/main/scala/org/locationtech/geomesa/tools/accumulo/commands/BinaryExportCommand.scala

```diff
@@ -0,0 +1,63 @@
+/***********************************************************************
+  * Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
```

Comment:
`mvn license:format` should fix it - but double check, sometimes it will create two headers